src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -50,27 +50,35 @@ namespace Stratis.Bitcoin.Features.Miner
     /// </remarks>
     public class PosMinting
     {
-        public class StakeOutput
-        {
-            public UtxoStakeDescription UtxoStakeDescription;
-            public int Depth;
-        }
-
+        /// <summary>
+        /// Information related to UTXO that is required for staking.
+        /// </summary>
         public class UtxoStakeDescription
         {
+            /// <summary>Block's hash.</summary>
             public uint256 HashBlock;
+            /// <summary>UTXO.</summary>
```

Comment:
i'd very much like empty line before each new XMLDOC comment of unrelated properties/fields
the only exception where i tend NOT to have an empty line before the new XMLDOC is where i have a private field and then a property with getter that returns that field, like 

```
/// <summary>..something..</summary>
private int x;
/// <summary>..something..</summary>
public int X { get { return x; } };
```
in which case it feels that those two guys are so tightly coupled that they don't deserve an empty space

also all these guys should be properties according to our coding style -> add { get; set; } to all of them